extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -403,7 +403,9 @@ describe('amp-a4a', () => {
         const doc = fixture.doc;
         const a4aElement = doc.createElement('amp-a4a');
         const a4a = new AmpA4A(a4aElement);
-        return a4a.maybeRenderAmpAd_(null).then(rendered => {
+        a4a.adUrl_ = 'http://foo.com';
+        a4a.maybeRenderAmpAd_ = function() { return Promise.resolve(false); }
```

Comment:
I don't think this is the right fix.  `maybeRenderAmpAd_()` _should_ be able to handle a `null` input.  That should cause `utf8FromArrayBuffer()` to throw an error.  It looks like the missing bit is that `utf8FromArrayBuffer()` is missing a `.catch()` clause.  (Alternatively, `maybeRenderAmpAd_()` could just check for `null` on input and return `Promise.resolve(false)` immediately.)

I could make that change if you like, or you could on this PR.  Happy either way.

What puzzles me is that this test _passed_ when I submitted the previous PR.  (Otherwise it wouldn't have gone in.)  Once again, I'm suspicious of the unit test framework's reliability.  :-(